src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/ActiveCampaignContactsWorkflow.cs

```diff
@@ -124,15 +126,19 @@ public override List<Exception> ValidateSettings()
             var list = new List<Exception>();
 
             if (string.IsNullOrEmpty(ContactMappings))
+            {
```

Comment:
This is the fix for [this issue](https://dev.azure.com/umbraco/D-Team%20Tracker/_sprints/taskboard/DXP%20team/D-Team%20Tracker/DXP%20Group%20Sprints/Cycle%2067%20-%20Sprint%202?workitem=46501)
Basically, without `else`, `var mappings = JsonSerializer.Deserialize<List<ContactMappingDto>>(ContactMappings);` will still be run, and would cause the error message body be empty on client side.